### PR TITLE
fix: ignore log zero buckets for NewSeries --story=135621641

### PR DESCRIPTION
**Pull request by chenguo367**
## Summary

- Ignore `bk_log_search:log` NewSeries data points with `value <= 0` before seen-set read/write and fire-map computation.
- Treat all-zero log buckets as a semantic empty batch for baseline completion, without writing those zero buckets into the seen-set.
- Keep non-log zero values observable, because zero can be a valid metric value.
- Add regression tests for log zero buckets, later real log buckets, cold-start baseline bootstrap, and metric zero values.

## Why

Log keyword data can receive synthetic zero buckets from date histogram completion. For NewSeries, those zero buckets should not create or consume a new-series fingerprint. The fix is scoped to the detector layer so query behavior for other callers remains unchanged.

## Validation

- `DJANGO_CONF_MODULE=conf.web.development.community BKAPP_SETTINGS_DEFAULT_CRONTAB='' BKAPP_SETTINGS_ACTION_TASK_CRONTAB='' BKAPP_SETTINGS_LONG_TASK_CRONTAB='' BKAPP_SETTINGS_EXCLUDE_WORKER_TASKS='' BKM_UNITTEST=1 uv run --project /Users/chenguo/projects/tencent/myapps/bk-monitor/bkmonitor python -m pytest alarm_backends/tests/service/detect/test_new_series.py -k 'log_count_zero_bucket or metric_zero_value' -q`
  - `4 passed, 24 deselected`
- `python3 -m py_compile bkmonitor/alarm_backends/service/detect/strategy/new_series.py bkmonitor/alarm_backends/tests/service/detect/test_new_series.py`
- `git diff --check`
- Commit hook: `ruff`, `ruff-format`, conflict/key/IP checks passed.

Full `test_new_series.py` is blocked in this local environment while creating the MySQL test database: `OperationalError: (1071, Specified key was too long; max key length is 3072 bytes)` on `ALTER TABLE apm_web_apmmetaconfig MODIFY level_key varchar(512) NOT NULL`. The four new pure algorithm tests run before that DB setup path and pass.

### CODE REVIEW

File: bkmonitor/alarm_backends/tests/service/detect/test_new_series.py [hunk 1]
```diff
@@ -93,6 +114,77 @@ def learn_start_exists(item):
     return key.NEW_SERIES_LEARN_START_KEY.client.exists(learn_key) == 1
 
 
+def use_fake_cache_router():
```
Comment (jayjiahua) [inline bkmonitor/alarm_backends/tests/service/detect/test_new_series.py:129]: **`bkmonitor/alarm_backends/tests/service/detect/test_new_series.py:117`** [P2]
> def use_fake_cache_router():

这里直接改写 `redis_cluster.DEFAULT_NODE`、`STRATEGY_ROUTER_CACHE` 和 `STRATEGY_NODE_MAP`，但没有在用例结束后恢复；pytest 同一进程中的后续用例会继承该 fake router，可能形成顺序依赖或掩盖真实路由问题。建议改成 `monkeypatch` fixture（或使用 yield fixture 保存并恢复旧值），让每个用例结束后自动回收全局状态。
